src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -117,6 +117,18 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         <button id = "sendButton" type="submit">Send</button>
     </form>
     
+    <span id = "key">
+      <p>
+      <center>
```

Comment:
You don't actually have to do anything here because we're focused on the Java side anyway, but note that in real life you probably wouldn't want to use the `<center>` tag. Instead, you'd probably use CSS and the `text-align` property to set the style. But this is okay for our purposes! :cat:

Also you might consider linking to the commonmark help page: http://commonmark.org/help/